packages/storybook-addon-designs/src/index.ts

```diff
@@ -1,21 +1,19 @@
-import { addons, makeDecorator, StoryWrapper } from "@storybook/addons";
+import { addons, makeDecorator } from "@storybook/preview-api";
```

Comment:
Minor nit but we shouldn't use `makeDecorator`. It only exists for some very legacy reason from Storybook 4.x and doesn't do anything relevant to modern versions of Storybook. Can be fixed later though.

```suggestion
import { addons, makeDecorator } from "@storybook/preview-api";
```

Reply:
Removed the decorator since this will be a new major version of the addon and was easy to do here.